twitter_ads/active_entities.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 Twitter, Inc.
+
+"""Container for all active entities logic used by the Ads API SDK."""
+
+from twitter_ads.enum import TRANSFORM
+from twitter_ads.resource import resource_property, Resource
+
+
+class ActiveEntities(Resource):
+
+    PROPERTIES = {}
+
+    # TODO: use `API_VERSION`
```

Comment:
I know this is listed as a TODO but could you make the change to `API_VERSION`? We can assume this won't be merged until #179 is